### PR TITLE
Fixed to output html on Windows avoiding print() function

### LINKED ISSUE #87: Hyper CLI script fails to load HTML on Windows
Observe:

```
C:\Users\cb2\Documents\hyper>hyper GET https://http2bin.org/html
Traceback (most recent call last):
  File "C:\Python34\Scripts\hyper-script.py", line 9, in <module>
    load_entry_point('hyper==0.2.0', 'console_scripts', 'hyper')()
  File "c:\users\cb2\documents\hyper\hyper\cli.py", line 227, in main
    print(request(args))
  File "C:\Python34\lib\encodings\cp850.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_map)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\u2014' in position
1266: character maps to <undefined>
```

### PR DESCRIPTION
**Pull request by t2y**
Fix #87

### CODE REVIEW

**Comment by Lukasa** [comment]
application/json does not allow charsets, so we should remove that.

Additionally, JSON must always be encoded in one of UTF-8, UTF-16 and UTF-32.

**Comment by t2y** [comment]
> application/json does not allow charsets, so we should remove that.

Oh, I didn't know that. Sorry, I'll fix it.

**Comment by t2y** [comment]
I noticed we also should handle the difference between py27 and py3.

**Comment by Lukasa** [comment]
@t2y No reason to apologise, most people don't know it. =)

**Comment by t2y** [comment]
It works well on OS X. I hope so on Windows, too ...

* python3.4
```bash
(hyper)$ hyper POST https://nghttp2.org/httpbin/post name=はいぱー version=2.0
{'args': {},
 'data': '{"name": "\\u306f\\u3044\\u3071\\u30fc", "version": "2.0"}',
 'files': {},
 'form': {},
 'headers': {'Connection': 'close',
             'Content-Length': '54',
             'Content-Type': 'application/json; charset=utf-8',
             'Host': 'httpbin',
             'Via': '2.0 nghttpx'},
 'json': {'name': 'はいぱー', 'version': '2.0'},
 'origin': '153.173.23.203',
 'url': 'https://httpbin/post'}
```

* python2.7
```bash
(hyper-py27)$ hyper POST https://nghttp2.org/httpbin/post name=はいぱー version=2.0
{u'args': {},
 u'data': u'{"version": "2.0", "name": "\\u306f\\u3044\\u3071\\u30fc"}',
 u'files': {},
 u'form': {},
 u'headers': {u'Connection': u'close',
              u'Content-Length': u'54',
              u'Content-Type': u'application/json; charset=utf-8',
              u'Host': u'httpbin',
              u'Via': u'2.0 nghttpx'},
 u'json': {u'name': u'\u306f\u3044\u3071\u30fc', u'version': u'2.0'},
 u'origin': u'153.173.23.203',
 u'url': u'https://httpbin/post'}
```

File: hyper/cli.py [hunk 1]
```diff
@@ -160,13 +161,16 @@ def set_request_data(args):
...
+            if is_py2:  # pragma: no cover
+                value = value.decode(PREFERRED_ENCODING)
+            body[i.key] = value
 
     if params:
         args.url.path += '?' + urlencode(params)
 
     if body:
-        content_type = 'application/json; charset=%s' % FILESYSTEM_ENCODING
+        content_type = 'application/json; charset=utf-8'
```
Comment (Lukasa) [inline hyper/cli.py]: There should be no charset at all. =) `content-type = 'application/json`.

**Comment by Lukasa** [comment]
One small note, but the function is otherwise correct. =)

File: hyper/cli.py (hunk 1 above)
Comment (sigmavirus24) [inline hyper/cli.py]: For reference: https://tools.ietf.org/html/rfc7159#section-8.1 and https://tools.ietf.org/html/rfc7159#section-11. You'll note in the latter that there are no optional parameters (which is what `charset` is).

Comment (t2y) [inline hyper/cli.py]: Thank you for the references. I learned a lot. Originally, I read httpie's client and I tried to tell that, but it has already been fixed. :smile: 
https://github.com/jakubroztocil/httpie/commit/6e7e2f2eeaf5d07060303735d8f183e526447f25

**Comment by Lukasa** [comment]
Hurrah, perfect! :cake:

**Pull request merged**